src/v2/components/MessageInput/MessageInput.tsx

```diff
@@ -105,7 +107,6 @@ type MessageInputPropsWithContext<
     | 'editing'
     | 'FileUploadPreview'
     | 'fileUploads'
-    | 'focused'
```

Comment:
if we're no longer using this should we remove it from the `MessageInputContext`?

Reply:
I meant to look if it was elsewhere...I will look and if it's orphan'd remove it.